presto-main/src/main/java/com/facebook/presto/metadata/FunctionManager.java

```diff
@@ -129,17 +131,23 @@ public FunctionManager(TypeManager typeManager, BlockEncodingSerde blockEncoding
         this(typeManager, createTestTransactionManager(), blockEncodingSerde, featuresConfig, new HandleResolver());
     }
 
-    public void loadFunctionNamespaces(String functionNamespaceManagerName, List<String> catalogSchemaPrefixes, Map<String, String> properties)
+    public void loadFunctionNamespaceManager(
+            String functionNamespaceManagerName,
+            String functionNamespaceManagerId,
+            List<String> catalogSchemaPrefixes,
+            Map<String, String> properties)
     {
-        requireNonNull(functionNamespaceManagerName, "connectorName is null");
```

Comment:
You probably want to keep this? Do change the `connectorName` though, seems like a copy-paste error 😝